modules/client-configuration/pages/clients-ubuntu.adoc

```diff
@@ -332,3 +334,11 @@ mgr-create-bootstrap-repo --with-custom-channels
 ----
 
 For more information on registering your clients, see xref:client-configuration:registration-overview.adoc[].
+
+
+[WARNING]
```

Comment:
Same as Debian. I am not aware about this being a problem for any supported Ubuntu version. This can go away.